index.js

```diff
@@ -0,0 +1,53 @@
+const list = ['掃除', '買い物', '散歩'];
+const display_text = `現在持っているタスク一覧\n`;
+function list_display() {
+  console.log(display_text);
+  for (let i = 0; i < list.length; i++) {
+    console.log(`${i}:${list[i]}`);
+  }
+};
+
+function list_add(task) {
+  list.push(task);
+}
+
+list_display();
+
+let i = 2;
+list_display();
+while (true) {
+  i++;
+  const task = prompt('タスクを登録してください');
+  list_add(task);
+  const added = alert("新しいタスクを追加しました");

```

Comment:
`added` 変数は使われていないので不要かと思いました。

```suggestion
  alert("新しいタスクを追加しました");
```